bounties/level-2/sep10-javascript-client.md

```diff
@@ -0,0 +1,63 @@
+# JavaScript SEP-0010 Client Reference Implementation
+
+## Award
+| Hunter | Reviewer | Type
+| :- | :- | :-
+| 800 XLM | 800 XLM | Capped (3)
+
+[//]: # (make sure to replace the file-name placeholders `BOUNTY_FILE_NAME_NO_EXTENSION` and `BOUNTY_FILE_NAME_WITH_EXTENSION` in the next two lines with the actual bounty filename)
+📜&nbsp; View [existing submissions](https://github.com/tyvdh/stellar-quest-bounties/issues?q=is%3Aissue+label%3A%3Csep10-javascript-client%3E) for this bounty. \
+🔵&nbsp; Start [hunting](https://github.com/tyvdh/stellar-quest-bounties/issues/new?assignees=&labels=&template=begin-the-hunt.yml&link=https://github.com/tyvdh/stellar-quest-bounties/blob/main/bounties/level-%3C2%3E/%3Csep10-javascript-client.md%3E) this bounty.
```

Comment:
The `<` and `>` around the placeholder need to as well. I'll fix the template (#102) as it is not obvious in the current state.

```suggestion
[//]: # (make sure to replace the file-name placeholders `BOUNTY_FILE_NAME_NO_EXTENSION` and `BOUNTY_FILE_NAME_WITH_EXTENSION` in the next two lines with the actual bounty filename)
📜&nbsp; View [existing submissions](https://github.com/tyvdh/stellar-quest-bounties/issues?q=is%3Aissue+label%3Asep10-javascript-client) for this bounty. \
🔵&nbsp; Start [hunting](https://github.com/tyvdh/stellar-quest-bounties/issues/new?assignees=&labels=&template=begin-the-hunt.yml&link=https://github.com/tyvdh/stellar-quest-bounties/blob/main/bounties/level-2/sep10-javascript-client.md) this bounty.
```